requirements.txt

```diff
@@ -0,0 +1,6 @@
+setuptools<=60.9
```

Comment:
The requirements to build the project (this same list) and to run it (only numpy) are already included in the `pyproject.toml` file. To avoid duplicated declarations, I would remove the `requirements.txt` file.